core/sup_tree_core/gear_config_poller.ex

```diff
@@ -29,8 +33,17 @@ defmodule AntikytheraCore.GearConfigPoller do
     {:ok, %{last_checked_at: 0}, @interval}
   end
 
+  @impl true
+  def handle_cast(:reload, state) do
+    handle_gear_config_loading(state)
+  end
+
   @impl true
   def handle_info(:timeout, state) do
+    handle_gear_config_loading(state)
+  end
+
+  defunp handle_gear_config_loading(state :: state_t) :: state_t do
```

Comment:
Now the return type isn't `state_t`; it should be `{:noreply, state_t, timeout}`.

Reply:
Oops, I had overlooked.
fixed in 9a7571c